src/main/main.go

```diff
@@ -16,10 +16,20 @@ func main() {
 	config.ProcessArgs()
 	logger := utils.InitLogger(config.GlobalConfig.Verbose)
 	logger.Infof("Assisted installer started. Configuration is:\n %+v", config.GlobalConfig)
+
+	ops := ops.NewOps(logger)
+	out, err := ops.ExecPrivilegeCommand(nil, "cat", config.GlobalConfig.PullSecretPath)
+	if err != nil {
+		logger.Warnf("Failed to Read Pull Secret file: %v", err)
```

Comment:
Should this be just a warning, or an error which will fail the installer?

Reply:
It is aligned with Agent. Only Warn for now